splunk_eventgen/eventgen_core.py

```diff
@@ -78,12 +56,11 @@ def __init__(self, args=None):
         self._generator_queue_size = getattr(self.args, 'generator_queue_size', 500)
         if self._generator_queue_size < 0:
             self._generator_queue_size = 0
-        self.logger.info("set generator queue size to %d", self._generator_queue_size)
+        self.logger.info("Set generator queue size to %d" % self._generator_queue_size)
```

Comment:
we really should convert this to "".format, this isn't supported in py3

Reply:
Changed.